libs/@local/effect-dns/hickory/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+edition = "2021"
+name    = "local_effect-dns-hickory"
+version = "0.0.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+napi        = { version = "2.16.15", default-features = false, features = ["napi9", "tokio_rt"] }
+napi-derive = "2.16.13"
+
+[build-dependencies]
+napi-build = "2.1.4"
+
+[package.metadata.sync.turborepo]
+ignore = true
```

Comment:
Why we need to ignore it?

Reply:
Because the `package.json` isn't auto-generated. Otherwise the name would be rewritten as `@rust/...` and dependencies would be lost.